src/components/JobColumn.vue

```diff
@@ -28,9 +28,9 @@
         v-for="(job, index) in jobs" :key="job.id"
         :jobId="job.id"
         :jobType="job.type"
-        :createdAt="job.created_at"
-        :updatedAt="job.updated_at"
-        :startedAt="job.started_at"
+        :createdAt="format(job.created_at, 'DD/MM/YYYY HH:mm:ss')"
```

Comment:
We can't access the **format** function here, directly in the template. We must "bind" an imported function to a variable in the _Vue reactive scope_ or create an another function as a wrapper.